packages/anoncreds/tests/InMemoryAnonCredsRegistry.ts

```diff
@@ -0,0 +1,157 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
Should this be here or is it just leftover?

Reply:
the agentContext is not used anywhere, so it's still needed